doc/nrf/known_issues.rst

```diff
@@ -122,7 +123,7 @@ IRIS-2676: Missing support for FOTA on nRF Cloud
 Asset Tracker v2
 ================
 
-.. rst-class:: v2-2-0
+.. rst-class:: v2-3-0 v2-2-0
```

Comment:
```suggestion
.. rst-class:: v2-2-0
```

No longer getting the build error. I also tried on 2.2.0 and it succeeded which is weird.
@gregersrygg Could you try to build with `overlay-carrier.conf` on the affecting versions?

Reply:
I could build atv2 for thingy:91 with `overlay-carrier.conf` from `main`, but `v2.2-branch` failed for me with the same error as before:

    Error: Image size (0x68a34) + trailer (0x630) exceeds requested size 0x69000

Reply:
Alright, thanks! @greg-fe then the suggestion is valid. Remove the entry for 2.3.0.